### DISCUSSION TITLE
Incorrect model type selected for depth image

### DISCUSSION THREAD

**Discussion by gerroon** [carson-katri/dream-textures#821] [Q&A] [answered]
Hi

Which model am I supposed to download for the viewport projections? thanks

**Comment by carson-katri** [comment, marked as answer]
You can use any normal model you want if you enable the ControlNet depth option and use a depth ControlNet (`sd-controlnet-depth`).

Otherwise, you'll need a depth model such as `stable-diffusion-2-depth`
